openpype/modules/deadline/plugins/publish/submit_maya_deadline.py

```diff
@@ -863,65 +467,38 @@ def _get_maya_payload(self, data):
             if int(rman_version.split(".")[0]) > 22:
                 renderer = "renderman22"
 
-        plugin_info = {
-            "SceneFile": data["filepath"],
+        plugin_info = copy.deepcopy(self.plugin_info)
+        plugin_info.update({
             # Output directory and filename
             "OutputFilePath": data["dirname"].replace("\\", "/"),
-            "OutputFilePrefix": data["render_variables"]["filename_prefix"],  # noqa: E501
+            "OutputFilePrefix": layer_prefix,
+        })
 
-            # Only render layers are considered renderable in this pipeline
-            "UsingRenderLayers": True,
+        return job_info, plugin_info
 
-            # Render only this layer
-            "RenderLayer": data["renderlayer"],
+    def _get_vray_export_payload(self, data):
 
-            # Determine which renderer to use from the file itself
-            "Renderer": renderer,
+        job_info = copy.deepcopy(self.job_info)
 
-            # Resolve relative references
-            "ProjectPath": data["workspace"],
-        }
-        payload["JobInfo"].update(job_info_ext)
-        payload["PluginInfo"].update(plugin_info)
-        return payload
+        job_info.Name = self._job_info_label("Export")
 
-    def _get_vray_export_payload(self, data):
-        payload = copy.deepcopy(self.payload_skeleton)
+        # Get V-Ray settings info to compute output path
         vray_settings = cmds.ls(type="VRaySettingsNode")
         node = vray_settings[0]
         template = cmds.getAttr("{}.vrscene_filename".format(node))
         scene, _ = os.path.splitext(data["filename"])
         first_file = self.format_vray_output_filename(scene, template)
         first_file = "{}/{}".format(data["workspace"], first_file)
         output = os.path.dirname(first_file)
-        job_info_ext = {
-            # Job name, as seen in Monitor
-            "Name": "Export {} [{}-{}]".format(
-                data["jobname"],
-                int(self._instance.data["frameStartHandle"]),
-                int(self._instance.data["frameEndHandle"])),
-
-            "Plugin": self._instance.data.get(
-                "mayaRenderPlugin", "MayaPype"),
-            "FramesPerTask": self._instance.data.get("framesPerTask", 1)
-        }
 
-        plugin_info_ext = {
-            # Renderer
+        plugin_info = {
```

Comment:
just note that we should probably move all those to the similar class like `MayaPluginInfo`

Reply:
These are actually PluginInfo for different plug-ins across the file. So this is one for the Tile Assembler, others I believe are for VRay plugin and Arnold plugin. But yes, it'd be more consistent if each would get their own plugin info accordingly.

Reply:
I've now refactored them to their own PluginInfo data classes - I wonder if that's more readable now. It feels a lot cleaner like this.